### PR TITLE
Set AWS_S3_MAX_MEMORY_SIZE

### PR DESCRIPTION
**Pull request by danihodovic**
> The maximum amount of memory (in bytes) a file can take up before being
> rolled over into a temporary file on disk.

Prevents out of memory eviction when a large file is uploaded.

### CODE REVIEW

**Review by browniebroke — APPROVED** [review]

File: {{cookiecutter.project_slug}}/config/settings/production.py [hunk 1]
```diff
@@ -88,6 +88,8 @@
     "CacheControl": f"max-age={_AWS_EXPIRY}, s-maxage={_AWS_EXPIRY}, must-revalidate"
 }
 # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
+AWS_S3_MAX_MEMORY_SIZE = env.int("AWS_S3_MAX_MEMORY_SIZE", default=100_000_000)  # 100MB
```
Comment (luzfcb) [inline {{cookiecutter.project_slug}}/config/settings/production.py:91]: To follow our standard, it probably needs to change to `DJANGO_AWS_S3_MAX_MEMORY_SIZE` and also update the settings table in the docs: https://github.com/cookiecutter/cookiecutter-django/blob/master/docs/settings.rst

```suggestion
AWS_S3_MAX_MEMORY_SIZE = env.int("DJANGO_AWS_S3_MAX_MEMORY_SIZE", default=100_000_000)  # 100MB
```

**Review by luzfcb — CHANGES_REQUESTED** [review]

**Comment by danihodovic** [comment]
Amended fixes. Would you like me to rebase and squash?

File: {{cookiecutter.project_slug}}/config/settings/production.py [hunk 2]
```diff
@@ -88,6 +88,8 @@
     "CacheControl": f"max-age={_AWS_EXPIRY}, s-maxage={_AWS_EXPIRY}, must-revalidate"
 }
 # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
+AWS_S3_MAX_MEMORY_SIZE = env.int("DJANGO_AWS_S3_MAX_MEMORY_SIZE", default=100_000_000)  # 100MB
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/config/settings/production.py:91]:
```suggestion
AWS_S3_MAX_MEMORY_SIZE = env.int(
    "DJANGO_AWS_S3_MAX_MEMORY_SIZE", 
    default=100_000_000,    # 100MB
)
```

File: {{cookiecutter.project_slug}}/config/settings/production.py [hunk 3]
```diff
@@ -88,6 +88,11 @@
     "CacheControl": f"max-age={_AWS_EXPIRY}, s-maxage={_AWS_EXPIRY}, must-revalidate"
 }
 # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
+AWS_S3_MAX_MEMORY_SIZE = env.int(
+    "DJANGO_AWS_S3_MAX_MEMORY_SIZE", 
+    default=100_000_000,    # 100MB
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/config/settings/production.py:93]:
```suggestion
    default=100_000_000,  # 100MB
```

File: {{cookiecutter.project_slug}}/config/settings/production.py [hunk 4]
```diff
@@ -88,6 +88,11 @@
     "CacheControl": f"max-age={_AWS_EXPIRY}, s-maxage={_AWS_EXPIRY}, must-revalidate"
 }
 # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
+AWS_S3_MAX_MEMORY_SIZE = env.int(
+    "DJANGO_AWS_S3_MAX_MEMORY_SIZE",
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/config/settings/production.py:92]:
```suggestion
    "DJANGO_AWS_S3_MAX_MEMORY_SIZE",
```

**Review by browniebroke — APPROVED** [review]
I think that's good to go now @luzfcb?

**Comment by danihodovic** [comment]
Ping @luzfcb

**Comment by browniebroke** [comment]
I'm going to go ahead and assume that it's fine. If not, we can always revert this. Thanks for the contribution @danihodovic!

**Pull request merged**